components/webui/client/src/SqlParser/index.ts

```
@@ -0,0 +1,46 @@
+import {
+    CharStream,
+    CommonTokenStream,
+    ErrorListener,
+    Recognizer,
+} from "antlr4";
+
+import SqlBaseLexer from "./generated/SqlBaseLexer";
+import SqlBaseParser from "./generated/SqlBaseParser";
+
+
+class SyntaxError extends Error {
```

Biome (2.1.2)

[error] Do not shadow the global "SyntaxError" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)